pkg/services/k8s/apiserver/restoptionsgetter.go

```diff
@@ -1,15 +1,14 @@
 package apiserver
 
 import (
-	"github.com/grafana/grafana/pkg/registry/corekind"
-	"github.com/grafana/kindsys"
 	"path"
 	"time"
 
+	"github.com/grafana/kindsys"
+
```

Comment:
* glares in newlines *

Reply:
@mildwonkey vscode must have been moving them around. deleting all newlines and re-saving seems to have fixed it